packages/graphql/lib/src/core/query_manager.dart

```diff
@@ -219,7 +219,14 @@ class QueryManager {
   }
 
   Future<QueryResult> refetchQuery(String queryId) {
-    final WatchQueryOptions options = queries[queryId].options;
+    WatchQueryOptions options = queries[queryId].options;
+    if (options.fetchPolicy == FetchPolicy.cacheFirst) {
+      options = options.copy()
+        ..policies = Policies(
+          fetch: FetchPolicy.cacheAndNetwork,
```

Comment:
Shouldn't it be FetchPolicy.networkOnly, as it always need to fetch the data from the server. How you changed it it will still return from cache?

Reply:
@Sh1d0w  Hey, what I found in [here](https://github.com/zino-app/graphql-flutter/blob/42d010c177f7fa752b9a0b5ca57548dd1cd3ad8f/packages/graphql/lib/src/core/query_options.dart#L33) that won't stop network fetching, and `cacheAndNetwork` won't affect the cache of origin fetch policy `cacheFirst`